Source/WebCore/platform/libwpe/PlatformPasteboardLibWPE.cpp

```diff
@@ -84,23 +80,21 @@ String PlatformPasteboard::readString(size_t, const String& type) const
 
 void PlatformPasteboard::write(const PasteboardWebContent& content)
 {
-    static const char plainText[] = "text/plain;charset=utf-8";
-    static const char htmlText[] = "text/html;charset=utf-8";
+    static const auto plainText = "text/plain;charset=utf-8"_s;
+    static const auto htmlText = "text/html;charset=utf-8"_s;
```

Comment:
These could even be `constexpr`:

```suggestion
    static constexpr auto plainText = "text/plain;charset=utf-8"_s;
    static constexpr auto htmlText = "text/html;charset=utf-8"_s;
```